Source/TheLudoGameApp/Classes/MainMenu.cs

```diff
@@ -0,0 +1,64 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Text;
+
+namespace TheLudoGameApp.Classes
+{
+    class MainMenu
+    {
+        public static void Menu()
+        {
+            GameMenu menu = new GameMenu();
+
+            bool mainMenu = true;
```

Comment:
the name mainMenu does not reveal what this bool contains.
Boolans should have an affirmative phrase and it should also have a name so that the boolean value makes sense , like `isMainMenuShown`
https://docs.microsoft.com/en-us/dotnet/standard/design-guidelines/names-of-type-members